robber/matchers/numbers.py

```diff
@@ -34,10 +34,35 @@ def matches(self):
     def failure_message(self):
         return 'Expected %g to be within %g and %g' % (self.actual, self.expected, self.args[0])
 
+
+class Change(Base):
+    def __init__(self, callable, obj=None, *args):
+        self.callable = callable
+        self.obj = obj
+        self.args = args
+        self.message = None
+
+    def match(self):
+        return self
+
+    def by(self, amount=0):
+        changed = self.callable(self.obj) - self.obj
```

Comment:
I wonder if `expect(callable).to.change(something).by(xxx)` could be anything else @steadykhoi?